docs/Reference/Rest_API/Rest.md

```diff
@@ -68,15 +68,15 @@ The [validator client API methods](https://ethereum.github.io/keymanager-APIs/)
 Enable the validator client API service from the command line by setting the
 [`--validator-api-enabled`](../CLI/CLI-Syntax.md#validator-api-enabled) command line option to `true`.
 
-You can also interact with APIs using Swagger UI by enabling the API documentation endpoint by setting the
+You can also interact with APIs by enabling the API documentation endpoint by setting the
 [`--validator-api-docs-enabled`](../CLI/CLI-Syntax.md#validator-api-docs-enabled) option to `true`.
 
-Access the APIs at `http:<interface>:<port>/swagger-ui` where:
+Access the APIs at `http:<interface>:<port>/swagger-docs` where:
```

Comment:
So the documentation for the APIs is available at swagger-docs, but its a huge mess of JSON that can really only be consumed by something...
`The openapi json document can be consumed from /swagger-docs if --validator-open-api-docs-enabled`?